tools/ci/start.sh

```diff
@@ -28,3 +28,4 @@ then
 fi
 
 sudo Xvfb $DISPLAY -screen 0 ${SCREEN_WIDTH}x${SCREEN_HEIGHT}x${SCREEN_DEPTH} &
+sudo fluxbox &
```

Comment:
I think this should be `fluxbox -display $DISPLAY` although it's possible that with one display only running we happen to get the right one.

Reply:
I had this before and reverted it. If you want I can definitely add it back.